pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/housekeeping/handler/EmailNotificationHandler.java

```diff
@@ -89,6 +101,34 @@ public void handleMessage(NotificationMessage message) {
             }
         }
 
+        Collection<String> toEmailList = message.getDistributionList();
+
+        //skip emails for PlayWright generated users / Email deny list
+        if (!emailDenyPatterns.isEmpty()) {
+            //collect emails from actual distribution list (DL) that matches deny patterns and remove from original email DL.
+            //If no additional valid emails, return... else continue with filtered new DL.
+            Collection<String> denyEmailList = new HashSet<>();
+            for (String toAddress : toEmailList) {
+                for (String emailPattern : emailDenyPatterns) {
+                    Pattern pattern = Pattern.compile(emailPattern);
+                    Matcher matcher = pattern.matcher(toAddress);
```

Comment:
Per my earlier comment (https://github.com/broadinstitute/ddp-study-server/pull/2671#pullrequestreview-1615321477), still wondering if an email pattern like `@test.firecloud.org` will also match `@testXfirecloudXorg`. If so, do we care?

Reply:
I believe it does. It makes it invalid email to start with though.
I think we can tweak the pattern we setup in conf if needed by adding \\
Not too concerned about it for our use case for now